data/phoible-references.bib

```diff
@@ -42457,3 +42457,23 @@ @book{heath2015togokan
 	Title = {A Grammar of Togo Kan (Dogon language family, Mali)},
 	Url = {https://deepblue.lib.umich.edu/handle/2027.42/123066},
 	Year = {2015}}
+
+@article{rose2021mojenotrinitario.pdf,
+	title={Mojeño Trinitario},
+	volume={52},
+	DOI={10.1017/S0025100320000365},
+	number={3},
+	journal={Journal of the International Phonetic Association},
+	publisher={Cambridge University Press},
+	author={Rose, Françoise},
+	year={2022},
+	pages={562–580}}
```

Comment:
@Alessioryan -- in latex / bibtex we use "--" for the page range; it becomes an en dash. Please update.

Reply:
Done!